t/process-type-privileged-agent.t

```diff
@@ -43,6 +43,21 @@ add_block_preprocessor(sub {
 
         if v == "privileged agent" then
             ngx.log(ngx.WARN, "process type: ", v)
+
+            ngx.timer.at(0, function() 
+   
+                for i = 1, 4 do 
+                    local tcpsock = ngx.socket.tcp()
```

Comment:
aliginment

Reply:
for is the body of a function argument of ngx.timer.at， then need ident